main/src/main/resources/report.html

```diff
@@ -229,6 +229,7 @@ <h1>GTFS Schedule Validation Report</h1>
         <span th:text="${config.gtfsSource}"></span><span
                 th:text="${config.countryCode.isUnknown()} ? '. No country code was provided.' : ', with the country code: ' + ${config.countryCode} + '.'"></span>
         </br>
+        <span th:if="${is_different_date}" th:text="'The date and time used during validation was ' + ${config.zonedDateTime.toLocalDateTime()} + '.'"></span></p>
```

Comment:
The format for this date is different from the one when the validator was run, e.g. `2024-01-03 at 17:14:01 EST` vs `2020-01-02T12:34`.
We should consider makeing the format the same.

Reply:
Yes. I admit to noticing this and being a bit lazy. Largely it was because the existing format is a custom format. I like your suggestion later about changing that format, in which case it will be easy for them to use the same format.

Reply:
Now the date-for-validation is just a date while the run time is a date+time. So there's no mismatch. I would still like to move away from the custom format, but not in this PR since it no longer is an issue.